src/components/Users/Users.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import styles from './Users.module.css';
+
+const Users = (props) => (
+  <div>
+    {
+                props.users.map((user) => (
```

Comment:
Consider not mapping in the return statement.

Reply:
/fxd